<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>851</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>随机数</string>
         </property>
         <widget class="QListWidget" name="randomList">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>201</width>
            <height>471</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>冒泡排序</string>
         </property>
         <widget class="QListWidget" name="buboList">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>201</width>
            <height>471</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>快速排序</string>
         </property>
         <widget class="QListWidget" name="quickList">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>201</width>
            <height>461</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startBtn">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
